lib/types/beacon_chain/beacon_state.ex

```diff
@@ -131,6 +131,19 @@ defmodule Types.BeaconState do
     end)
   end
 
+  def decode_ex(%__MODULE__{} = map) do
+    map
+    |> Map.update!(:validators, &Aja.Vector.new/1)
+    |> Map.update!(:balances, &Aja.Vector.new/1)
+    |> Map.update!(:randao_mixes, &Aja.Vector.new/1)
+    |> Map.update!(:previous_epoch_participation, &Aja.Vector.new/1)
+    |> Map.update!(:current_epoch_participation, &Aja.Vector.new/1)
+    |> Map.update!(:latest_execution_payload_header, &Types.ExecutionPayloadHeader.decode/1)
```

Comment:
Isn't this covered by ssz_ex as well? This is just normal decoding + special decoding of the u256. If u256 is a special case it should be covered in ssz_ex.